src/patients/GeneralInformation.tsx

```diff
@@ -145,6 +157,16 @@ const GeneralInformation = (props: Props): ReactElement => {
               isEditable={isEditable}
             />
           </div>
+          <div className="col">
+            <SelectWithLabelFormGroup
+              name="bloodType"
+              label={t('patient.bloodType')}
+              options={bloodTypeOptions}
+              defaultSelected={bloodTypeOptions.filter(({ value }) => value === patient.bloodType)}
+              onChange={(values) => onFieldChange('bloodType', values[0])}
+              isEditable={isEditable}
+            />
+          </div>
```

Comment:
let's add a test in `GeneralInformation.test.tsx` that it renders this select component. Let's also add a test that if the value of the component changes, that it calls the onChange function with the correct data.

Reply:
@jackcmeyer Thank you for your feedback. I have added the test cases.